docs/tasks.md

```diff
@@ -142,6 +115,29 @@ applies to both.
 
 Below is an example of a Pipeline declaration that uses a `ClusterTask`:
```

Comment:
It's worth clarifying that there is no v1 ClusterTask but you can still reference a v1beta1 clustertask in a v1 pipeline, and that the cluster resolver syntax below can be used to reference any task, not just a clustertask